packs/pathfinder-monster-core/hryngar-sharpshooter.json

```diff
@@ -541,7 +541,7 @@
                 "containerId": null,
                 "damage": null,
                 "description": {
-                    "value": "<p>This shot deals no damage, but on a hit, the target must succeed at a @Check[type:reflex|dc:16] or be knocked @UUID[Compendium.pf2e.conditionitems.Item.Prone] and @UUID[Compendium.pf2e.conditionitems.Item.Immobilized] until it is freed with a successful DC 15 check to @UUID[Compendium.pf2e.actionspf2e.Item.Escape]. This check can be attempted either by the target or a creature adjacent to the target.</p>"
+                    "value": "<p>This shot deals no damage, but on a hit, the target must succeed at a @Check[type:reflex|dc:16] or be knocked @UUID[Compendium.pf2e.conditionitems.Item.Prone] and @UUID[Compendium.pf2e.conditionitems.Item.Immobilized] until it is freed with a successful DC 15 check to [[/act escape dc=15]]. This check can be attempted either by the target or a creature adjacent to the target.</p>"
```

Comment:
I don't think we are implementing `/act` in the system just yet

Reply:
Can you elaborate? Several `/act` actions, including escape, are supported in the current release.

Reply:
As discussed on Discord, we'll leave this in for now.